airbyte-integrations/connectors/source-google-analytics-v4/source_google_analytics_v4/source.py

```diff
@@ -119,6 +119,16 @@ def next_page_token(self, response: requests.Response) -> Optional[Mapping[str,
         if next_page:
             return {"pageToken": next_page}
 
+    def should_retry(self, response: requests.Response) -> bool:
+        if response.status_code == 400:
+            self.logger.info(f"{response.json()['error']['message']}")
+            self.raise_on_http_errors = False
+
```

Comment:
Why here? default value of the property raise_on_http_errors is True. And by idea CDK should generate an exception before. Please add a code comment about this and its reason

Reply:
Updated.